admin/utils.go

```diff
@@ -56,3 +56,28 @@ func GetOrDefault[T any](ptr *T, defaultValue T) T {
 	}
 	return defaultValue
 }
+
+func TimePtrToStringPtr(t *time.Time) *string {
```

Comment:
It will be good to add helpers to the docs.
Options:

a) Create new docs file called helpers containing some examples
b) cover those in contributing.md
c) Add some examples/tests for helpers.